ros/src/simulation/gazebo_simulator/vehicle/vehicle_model/urdf/vehicle.xacro

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<robot name="autoware_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find vehicle_model)/urdf/body.xacro" />
+  <xacro:include filename="$(find vehicle_model)/urdf/wheel.xacro" />
+  <xacro:include filename="$(find vehicle_model)/urdf/steering.xacro" />
+  <xacro:include filename="$(find velodyne_description)/urdf/HDL-32E.urdf.xacro"/>
+  <xacro:include filename="$(find velodyne_description)/urdf/VLP-16.urdf.xacro"/>
+  <xacro:include filename="$(find camera_description)/urdf/monocular_camera.xacro"/>
+  <xacro:include filename="$(find imu_description)/urdf/imu.xacro"/>
+
+  <xacro:arg name="gpu" default="false"/>
+
+  <xacro:property name="pi" value="3.1415926835897931"/>
+
+  <link name="base_link"/>
+
+  <!-- ================ vehicle ================ -->
+  <!-- load parameter -->
+	<xacro:property name="vehicle_info" value="${load_yaml('$(find vehicle_model)/config/vehicle_info.yaml')}"/>
+  <!-- set parameter -->
+	<xacro:property name="wheel_radius" value="${vehicle_info['wheel_radius']}"/>
+  <xacro:property name="wheel_width" value="${vehicle_info['wheel_width']}"/>
+  <xacro:property name="wheel_base" value="${vehicle_info['wheel_base']}"/>
+  <xacro:property name="wheel_tread" value="${vehicle_info['wheel_tread']}"/>
+  <xacro:property name="vehicle_mass" value="${vehicle_info['vehicle_mass']}"/>
+  <xacro:property name="vehicle_length" value="${vehicle_info['vehicle_length']}"/>
+  <xacro:property name="vehicle_width" value="${vehicle_info['vehicle_width']}"/>
+  <xacro:property name="vehicle_height" value="${vehicle_info['vehicle_height']}"/>
+  <xacro:property name="vehicle_mass_height" value="0.5"/>
+
+  <!-- body-->
+  <xacro:body_macro suffix="base_body"
+                    parent="base_link"
+                    wheel_base="${wheel_base}"
+                    vehicle_mass="${vehicle_mass}"
+                    vehicle_mass_height="${vehicle_mass_height}"
+                    vehicle_length="${vehicle_length}"
+                    vehicle_width="${vehicle_width}"
+                    vehicle_height="${vehicle_height}"/>
+  <joint name="base_footprint_joint" type="fixed">
```

Comment:
In current Autoware version, this `base_footprint` definition is `base_link`.

Reply:
This is the result of the following issue
https://github.com/CPFL/Autoware/issues/1881

Reply:
I discussed @hakuturu583. as a result,  we decided to return to the original specification. e6deb8c73e05a5479e5de856e1b0f3bffddbe9ea